collect_app/src/androidTest/java/org/odk/collect/android/utilities/FormIndexSavepointTest.java

```diff
@@ -17,69 +17,31 @@
 package org.odk.collect.android.utilities;
 
 import org.javarosa.core.model.FormIndex;
-import org.javarosa.core.model.instance.TreeReference;
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
-import org.odk.collect.android.application.Collect;
-import org.odk.collect.android.logic.FormController;
 import org.odk.collect.android.tasks.SaveFormIndexTask;
 import org.odk.collect.android.tasks.SaveToDiskTask;
 
 import java.io.File;
 
 import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.assertNotNull;
 
+// Verify that a FormIndex can be saved to and restored from a file
 @RunWith(MockitoJUnitRunner.class)
 public class FormIndexSavepointTest {
 
-    private File instancePath;
-    @Mock
-    private FormController formController;
-
-    @Before
-    public void setUp() {
-        instancePath = new File(Collect.INSTANCES_PATH + File.separator + "test.xml");
-        when(formController.getInstancePath()).thenReturn(instancePath);
-        Collect.getInstance().setFormController(formController);
```

Comment:
I asked the questions "what is this testing" and "how would it fail" and it seems like doing this setup only lets us test Java/Android methods. The only thing we end up doing is calling `getName` on `instancePath` which is not ODK code.